### ISSUE TITLE
Calendar locale binding?

### ISSUE THREAD

**Issue by davidgfolch** [primefaces/primeng#618]
Calendar is not changing locale then locale is changed.  Initial locale is loading ok.

`<p-calendar [(ngModel)]="selectedDate" [locale]="locale" showAnim="slideDown" [showIcon]="true"></p-calendar>`

```
    this.locale=this.localizations[this.appConfig.language.toLowerCase()];  //THIS WORKS
    this.appConfig.languageObservable.subscribe((lang:string) => {
      this.locale=this.localizations[lang.toLowerCase()];  //THIS EVENT WORKS, BUT CALENDAR DONT GET UPDATED
    });

```

**Comment by iguissouma** [comment]
I have the same issue. 
Any update on this issue?

@davidgfolch did you find any workaround for this issue?

**Comment by gekallergis** [comment]
Same here! :+1:

**Comment by Prontsevich** [comment]
I have the same issue!

**Comment by VictorDeBlas** [comment]
I've got the same problem...

**Comment by cagataycivici** [comment]
Fixed in 4.0.0-RC2.
